assets/styles/pages/squad.less

```diff
@@ -99,6 +100,14 @@ section {
   display: none;
   text-align: center;
 }
+.members {
```

Comment:
Looking at the page from here https://aurora-dev-pr-378.herokuapp.com/squad, I feel like the image could use a little bit of whitespace around it. What would you think about trying something like this:
```less
.members {
  margin: @margin-md;
  text-align: center;
  img {
  max-width: 600px
    width: 100%;
  }
}
```

And then you also wouldn't need the `@media (max-width: @screen-xs-max)` that follows it for the `img`. Let me know what you think.

Reply:
Thanks for the suggestion!